plugins/symbol-processor/src/test/kotlin/org/jetbrains/dataframe/ksp/DataFrameJdbcSymbolProcessorTest.kt

```diff
@@ -212,6 +212,63 @@ class DataFrameJdbcSymbolProcessorTest {
         result.successfulCompilation shouldBe true
     }
 
+    /**
+     * Test code is copied from test above.
+     */
+    @Test
+    fun `schema extracted via readFromDB method is resolved with db credentials from env variables`() {
+        val result = KspCompilationTestRunner.compile(
+            TestCompilationParameters(
+                sources = listOf(
+                    SourceFile.kotlin(
+                        "MySources.kt",
+                        """
+                @file:ImportDataSchema(
+                     "Customer",
+                     "$CONNECTION_URL",
+                     jdbcOptions = JdbcOptions("", "", extractCredFromEnv = true, tableName = "Customer")
```

Comment:
Is it expected here to use empty strings?

Reply:
Yes, for now, it's fine, especially for a few databases (something could be empty row)